File: src/main/java/com/baedal/store/application/command/GetStoreDetailCommand.java
package com.baedal.store.application.command;

import com.baedal.store.domain.model.StoreReviewSummary;
import java.time.LocalTime;
import java.util.List;
import lombok.Builder;
import lombok.Getter;

@Getter
@Builder
public class GetStoreDetailCommand {

  private Long storeId;

  private String name;

  private String title;

  private String content;

  private String address;

  private String pictureUrl;

  private String category;

  private LocalTime openTime;

  private LocalTime closeTime;

  private int deliveryAmount;

  private List<StoreReviewCommand> top10Reviews;

  private List<ProductInfoCommand> products;

  private double averageScore;
}

File: src/main/java/com/baedal/store/kotlin/application/command/GetStoreDetailCommand.kt
package com.baedal.store.kotlin.application.command

import java.time.LocalTime

data class GetStoreDetailCommand(
    val storeId: Long,
    val name: String,
    val title: String,
    val content: String,
    val address: String,
    val pictureUrl: String,
    val category: String,
    val openTime: LocalTime,
    val closeTime: LocalTime,
    val deliveryAmount: Int,
    val averageScore: Double
)
